Templates/Cards/cards.html

```diff
@@ -2,119 +2,567 @@
+                <li>
+                    <a href="/Templates/Avatars/avatar.html">
+                        <span class="smol-ico">
+                            <i class="fs-l fas fa-users"></i>
+                        </span>
+                        <span class="smol-title"> Avatars </span>
+                    </a>
+                </li>
+                <li>
+                    <a href="/Templates/Alerts/Alert.html">
+                        <span class="smol-ico">
+                            <i class="fs-l fas fa-exclamation-circle"></i>
+                        </span>
+                        <span class="smol-title"> Alert </span>
+                    </a>
+                </li>
+                <li>
+                    <a href="/Templates/Badges/Badge.html">
+                        <span class="smol-ico">
+                            <i class="fs-l fas fa-ribbon"></i>
+                        </span>
+                        <span class="smol-title"> Badges </span>
+                    </a>
+                </li>
+                <li>
+                    <a href="/Templates/Buttons/buttons.html">
+                        <span class="smol-ico">
+                            <i class="fs-l fas fa-sliders-h"></i>
+                        </span>
+                        <span class="smol-title"> Buttons </span>
+                    </a>
+                </li>
+                <li>
+                    <a href="/Templates/Cards/cards.html">
+                        <span class="smol-ico">
+                            <i class="fs-l far fa-id-badge"></i>
+                        </span>
+                        <span class="smol-title"> Card </span>
+                    </a>
+                </li>
+                <li>
+                    <a href="/Templates/Images/images.html">
+                        <span class="smol-ico">
+                            <i class="fs-l far fa-images"></i>
+                        </span>
+                        <span class="smol-title"> Images </span>
+                    </a>
+                </li>
+                <li>
+                    <a href="/Templates/Inputs/inputs.html">
+                        <span class="smol-ico">
+                            <i class="fs-l far fa-keyboard"></i>
+                        </span>
+                        <span class="smol-title"> Input </span>
+                    </a>
+                </li>
+            </ul>
+        </div>
+        <div class="btn-toggle-drawer">
+            <span class="ico closed-ico">
+                <i class="fas fa-chevron-circle-right"></i>
+            </span>
+            <span class="ico open-ico">
+                <i class="fas fa-chevron-circle-left"></i>
+            </span>
+        </div>
+    </div>
+
+    <div class="container">
+        <h1 class="pd-y-md mg-y-sm">Cards</h1>
+        <div class="contentArticle pd-y-sm">
+            Cards are a set of similar meaning or category-falling content hoders.
```

Comment:
similar meaning or category falling - this phrase doesn't sound good. Replace it with like Cards are used to display items of same type or category

Reply:
alright, i'll see to it